test/acceptance/features/steps/knative_serving.py

```diff
@@ -0,0 +1,39 @@
+import re
+from command import Command
+from openshift import Openshift
+from pyshould import should_not
+
+
+class Knative_Serving():
+
+    openshift = Openshift()
+    cmd = Command()
+
+    name = ""
+    namespace = ""
+
+    def __init__(self, name="knative-serving", namespace="knative-serving"):
+        self.name = name
+        self.namespace = namespace
+        self.knative_serving_yaml_template = '''
+---
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: {name}
+  namespace: {namespace}
+'''
+
+    def is_present(self):
+        cmd = f'oc get ks {self.name} -n {self.namespace}'
+        output, exit_status = self.cmd.run(cmd)
+        if exit_status != 0:
+            return False
+        return True
+
+    def create(self):
+        serving_output = self.openshift.oc_apply(self.knative_serving_yaml_template.format(name=self.name, namespace=self.namespace))
+        pattern = f'knativeserving.(serving|operator).knative.dev/{self.namespace}\\screated'
+        output_match = re.search(pattern, serving_output)
+        output_match | should_not.be_none.desc("Matched knative serving creation output")
+        return True
```

Comment:
is it possible to return `False` from this function?

Reply:
Can return False like in case of nodejs_application.py(install()), However, with the intent of failing fast, introduced should if pattern fails. Test has to Fail early and made not to continue if creation of knative serving is failed. Hence has to FAIL early in case of failure and hence used Should package.

Reply:
the caller of this method could check the obtained result and fail if the received value is `False`.

Reply:
`if re.search(pattern, serving_output):
            return True
        print(f"Pattern {pattern} did not match as creating knative serving yaml output is {serving_output}")
        return False`

Let me know if this is fine, Thanks